### ISSUE TITLE
2.5.8 Target Size (Minimum): equivalent, inline, and essential exceptions

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/wcag#3253]
While there is a lot of good stuff on the spacing exception, I notice the equivalent exception is not covered in any detail in the 2.5.8 Target Size (Minimum) Understanding text. The sentence "there is another control that can achieve the underlying function" seems to leave a lot of room for interpretation. Would enlisting all functions in a 6pt text block in the footer with inline links (themselves exempt via the inline exception) qualify as equivalent exception? Does it make a difference if the equivalent function is 

1. on the same page?
2. in a disclosure on the same page?
3. indirectly accessible via a link to an alternative version on another page?

The equivalent exception does not spell this out, and perhaps cannot. Not sure whether this has been discussed before at any length.

The inline exception probably also deserves some more text and illustrations too, likely focussing on helping to determine whether list structures are exempt or fall under the SC (there used to be some which I cobbled together less nicely as Patrick's work which are now gone, possibly because they no longer fitted). 

The essential exception could use a small text section with an image of a map with many location markers in one area.

Do people think we should cover the other exceptions better than in the current text?

**Comment by patrickhlauke** [comment]
> Do people think we should cover the other exceptions better than in the current Understanding text?

Yes, absolutely. The clearer we can be with understanding docs, the better

**Comment by jamieherrera** [comment]
To @detlevhfischer's point, the sentence "there is another control that can achieve the underlying function" opened a question of whether "control" in this SC is specific to "target" which is pointer-specific, or can an equivalent "control" to mean more broadly any interaction that cab achieve the underlying function, which may not require pointer. Example: a text input field as an equivalent alternative to a button that opens a calendar for a birthdate or future appointment, or text input as an equivalent alternative to increment / decrement targets. As a person with motor issues, I prefer text inputs over buttons most of the time when offered as an alternative; is that equivalent? In some cases I would say yes. Is that the intent of the SC "equivalent" exception? As worded, I don't know.  

While the current Understanding doc drives home the "touch" alternatives, a more explicit statement that "equivalent" refers to an alternative "target" (with link, not "control") that meets the SC minimums would reduce noise around alternatives that are not equivalent explicitly as a touch/pointer experience.

**Comment by patrickhlauke** [comment]
> As part of a larger research, I found that no, keyboard alternatives are not equivalent

of course they are equivalent if we're talking about controls - such as inputs etc. what is not acceptable are cases where you say "well, we also implemented shortcut keys, or users can just tab to these things and activate them with space/enter, so it's fine".
